hometask8/controllers/Groups.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+var url = require('url');
+
+
+var Groups = require('./GroupsService');
+
+
+module.exports.groupPost = function groupPost (req, res, next) {
+  var group = req.swagger.params['group'].value;
+  
+
+  var result = Groups.groupPost(group);
+
+  if(typeof result !== 'undefined') {
```

Comment:
The check "if(typeof result !== 'undefined')" is better to be replaced by the check "if (result)" as "undefined" is not the only "falsy" statement.